pkg/tenant/tenants.go

```diff
@@ -10,8 +10,29 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+// TenantList retrieves all tenants assigned to an organization and
+// returns a 200 OK response.
+//
+// Route: /tenant
 func (s *Server) TenantList(c *gin.Context) {
-	c.JSON(http.StatusNotImplemented, "not implemented yet")
+	// Get tenants from the database and return a 500 response if not successful.
+	var tenant *db.Tenant
+	if _, err := db.ListTenants(c.Request.Context(), tenant.OrgID); err != nil {
```

Comment:
The problem is that `tenant` (declared on line 19) is `nil` so when you try to do `tenant.OrgID` - you're trying to get the property of a `nil` struct, which is causing the panic. 

The `orgID` will come from the JWT credentials of the user trying to list the tenants; so authentication is needed before you can implement the "list filtering". For now, I think you should pass in an empty ULID (e.g. `ulid.ULID{}`) and add a TODO that the `orgID` must be fetched. The TenantList will then return _all_ tenants in the database; but you can write TODOs in the test to expect that the list is filtered so only the tenants for the logged in user are returned.